content/talks/code-reviews.md

```diff
@@ -16,7 +16,7 @@ events:
     date: 2015-11-11
     slides: https://www.slideshare.net/Funstacy/sass-code-reviews-how-one-code-review-changed-my-life-sassconf2015
   - venue: CSS Dev Conf
-    url: https://twitter.com/cssdevconf
+    url: http://2015.cssdevconf.com/
```

Comment:
@mirisuzanne Why this change? The link is broken for me, which is why I replaced it with the Twitter link.

Reply:
It works fine for me, so I'm confused.

Reply:
Weird. Apparently my FF has a cached redirect to https:// that I can't get rid of... it works in Chrome 👍